editor.Rmd

```diff
@@ -3,14 +3,52 @@
 # Review, Publish, Promote {#editor}
 
 ## Review a Post
-This section explains how to review a post.
+This section explains how to review a blog post or tech note.
 The editor for a post is typically the Community Manager or their intern.
 
 A post is submitted as a pull request on GitHub and we use the [well documented GitHub web interface](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request) to suggest edits, make comments, and ultimately merge the pull request to publish the post.
 
+### Review
+
+ADD screenshot(s) of review interface
```

Comment:
Do you want to do this before or after this is merged? What aspects exactly do you want to highlight? Maybe you should screencast the entire review process next time you review a post?